### PR TITLE
UI and UX improvements

### PR DESCRIPTION
**Pull request by baptistebisson**
I've made multiple commits so it's easier to review.

Mostly UI and UX to try to improve the design. Not sure if you're willing to have rounded borders.

### CODE REVIEW

File: src/routes/app/maps/create/+page.svelte [hunk 1]
```diff
@@ -33,6 +34,7 @@
         },
       });
     }
+    alertSuccess.set('Map `'+ form.name +'` created.');
```
Comment (calinvladth) [inline src/routes/app/maps/create/+page.svelte]: You can use `successHandler` from `utils/alertHandler`. There is some logic that clears the alert from the view.
By using `alertSuccess` directly, it works, but it's a bypass.

File: src/routes/app/+page.svelte [hunk 2]
```diff
@@ -22,11 +22,11 @@
     <Button on:click={() => goto(ROUTES.MAPS_CREATE)}>Create</Button>
   </div>
 
-  <h1 class="text-3xl p-5 border-b border-black">Maps</h1>
+  <h1 class="text-3xl p-5 border-b border-black">Maps{@html maps.length > 0 ? " <small>(" + maps.length + ")</small>": ""}</h1>
 
   {#if maps.length === 0}
     <div class="w-full h-full flex items-center justify-center">
-      <p>No maps available</p>
+      <p><a href="{ROUTES.MAPS_CREATE}" class="underline">Create your first map</a></p>
```
Comment (calinvladth) [inline src/routes/app/+page.svelte]: paragraph can be removed, just leave the anchor

File: src/routes/app/+page.svelte [hunk 3]
```diff
@@ -22,11 +22,11 @@
     <Button on:click={() => goto(ROUTES.MAPS_CREATE)}>Create</Button>
   </div>
 
-  <h1 class="text-3xl p-5 border-b border-black">Maps</h1>
+  <h1 class="text-3xl p-5 border-b border-black">Maps{@html maps.length > 0 ? " <small>(" + maps.length + ")</small>": ""}</h1>
```
Comment (calinvladth) [inline src/routes/app/+page.svelte]: Nitpicking, I think it's easier to read like this: 
`Maps {@html maps.length > 0 ? "<small>(" + maps.length + ")</small>": ""}`

**Review by calinvladth — CHANGES_REQUESTED** [review]

File: src/routes/app/+page.svelte (hunk 3 above)
Comment (baptistebisson) [reply src/routes/app/+page.svelte]: You only added space outside ?

File: src/routes/app/maps/create/+page.svelte (hunk 1 above)
Comment (baptistebisson) [reply src/routes/app/maps/create/+page.svelte]: Didn't see it, thanks

**Comment by baptistebisson** [comment]
I've pushed the modification. 

Had some issues on linux, pocketbase doesn't work, I had to download it.
Shouldn't we remove it from repo ? Pretty heavy (but too late now) and use a bash script to setup the project.

We should also use .env file to specify the connection port (default port is already taken for me)

File: src/routes/app/+page.svelte (hunk 3 above)
Comment (calinvladth) [reply src/routes/app/+page.svelte]: Yes

**Comment by calinvladth** [comment]
> I've pushed the modification.
> 
> Had some issues on linux, pocketbase doesn't work, I had to download it. Shouldn't we remove it from repo ? Pretty heavy (but too late now) and use a bash script to setup the project.
> 
> We should also use .env file to specify the connection port (default port is already taken for me)

This is a good idea, thank you for suggesting.

**Pull request merged**